extension/src/prototypes/view/selection/SpatialIdObjectContent.tsx

```diff
@@ -91,6 +93,40 @@ export const SpatialIdObjectContent: FC<SpatialIdObjectContentProps> = ({ values
     ];
   }, [features, values]);
 
+  const [files, setFiles] = useState<object | undefined>();
+  const [_loading, setLoading] = useState<boolean>(true);
+  const [_error, setError] = useState<string | null>(null);
+  const conditions = useMemo(() => {
+    const feature = features.find(feature => parseIdentifier(values[0]).key === feature.id);
+    if (!feature) return;
+    return `s:${feature.data.zfxyStr}`;
```

Comment:
What is `s:`?

Reply:
Updated with params base on API document.